packages/ui/src/routes/networks/NetworksPage.tsx

```diff
@@ -97,6 +41,116 @@ const NetworksPage = () => {
             },
         })
     }
+
+    const moveMainNetworkCard = useCallback(
+        (draggedIndex: number, hoveredOnIndex: number) => {
+            const draggedItem = mainNetworks[draggedIndex]
```

Comment:
This code is almost the same as the other useCallback, isn't there a way to avoid this duplication?

Reply:
I don't know maybe you can get the type (`mainnet` | `testnet`) as a function parameter

Reply:
Done 👌